flake.nix

```
@@ -40,6 +40,7 @@
             nodePackages.pnpm
             poetry
             jq
+            electron_30-bin
```

```suggestion
            electron_30-bin
            killall
```
just noticed that decky needs killall or it can't restart the ui :upside_down_face: 
could put it in another pr but meh you can sneak it in here ;)